src/constants.js

```diff
@@ -5,5 +5,7 @@ module.exports = {
     TVGUIDE_MAXIMUM_FLEX_DURATION : 6 * 60 * 60 * 1000,
     TOO_FREQUENT: 100,
 
-    VERSION_NAME: "1.3.0-prerelease"
+    VERSION_NAME: "1.3.0-prerelease",
+    HOST: undefined,
```

Comment:
If `HOST` and `DATABASE` are `undefined` here and we're setting them later, they're not really constants in the same way that these other numbers and strings in this file, right?